backend/apps/volontulo/tests/views/api/test_contact.py

```diff
@@ -0,0 +1,158 @@
+# pylint: disable=missing-docstring
+from unittest import mock
+
+from django.contrib.auth import get_user_model
+from django.test import TransactionTestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from apps.volontulo.factories import UserFactory
+from apps.volontulo.serializers import ContactSerializer
+
+ENDPOINT_URL = reverse('contact')
+
+
+def get_admin_email():
```

Comment:
Instead of function in global scope, we should add this to `setUp` or `setUpClass` method inside test case class that will use that.